kotlin/samples/containers/poetry/src/main/res/drawable/list_selector.xml

```diff
@@ -14,7 +14,7 @@
   ~ limitations under the License.
   -->
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-  <item android:state_activated="true">
+  <item android:state_selected="true">
```

Comment:
Espresso makes it easy to check for `selected` state, but not `activated`. `selected` seems like the more natural flag to use for this anyway.

Reply:
Does this have a visible effect, though?

Reply:
The property used in this selector just needs to match the one we're setting in the code – can't see a difference between `activated` and `selected` otherwise.